services/121-service/src/cronjob/cronjob.service.ts

```diff
@@ -9,6 +10,18 @@ function shouldBeEnabled(envVariable: string | undefined): boolean {
   return !!envVariable && envVariable.toLowerCase() === 'true';
 }
 
+// ##TODO: See if we can change this function to getCronFunctions and just get the public functions with the @Cron decorator
+function getAllMethods(instance: any): string[] {
```

Comment:
Why did you chose to use a seperate function here? Could this logic not be in  public getAllMethods()

Reply:
I could not get that to work before, I will try again.

Reply:
Solved.